### PR TITLE
Adding note about proxies

### PR DESCRIPTION
**Pull request by victoriaspek**
@salsakran Started by adding this note, I don't know enough about proxies to know if there's more general advice we can give, let me know if there's more we can add.

### CODE REVIEW

File: docs/enterprise-guide/activating-the-enterprise-edition.md [hunk 1]
```diff
@@ -2,12 +2,18 @@
...
 
 To get the Enterprise edition, you can either [download the latest .jar file](https://downloads.metabase.com/enterprise/latest/metabase.jar), or get the [latest Docker image](https://hub.docker.com/r/metabase/metabase-enterprise/) at `metabase/metabase-enterprise:latest`
 
 Once you have the Enterprise edition running, to activate all of its features go to the Admin Panel within Metabase, click on the Enterprise tab, click the "Activate a license" button, and then paste in your license token. The page should change to show you that Enterprise features are now active.
 
+Please note that if you are running Metabase via a proxy, you may need to complete some additional configuration steps. Metabase will need to be able to access `https://store.metabase.com/api/[token-id]` in order to validate your token and turn on the Enterprise feature set. This may include unblocking ports for Metabase to make an outgoing connection. If you're unsure of how to do this, please contact your organization's ops or infrastructure team.
```
Comment (salsakran) [inline docs/enterprise-guide/activating-the-enterprise-edition.md]: minor wording nit to pick : it's not a matter of "running metabase via proxy", it's whether the Metabase server can make requests to the internet (specifically store.metabase.com as you mention) or whether all outbound traffic needs to be routed to the proxy.

if the latter is true, you would do this via the command below.

**Review by salsakran — CHANGES_REQUESTED** [review]
a bit of rewording would make things a bit clearer.

**Review by salsakran — APPROVED** [review]

File: docs/enterprise-guide/activating-the-enterprise-edition.md [hunk 2]
```diff
@@ -2,12 +2,22 @@
...
 
 Once you have the Enterprise edition running, to activate all of its features go to the Admin Panel within Metabase, click on the Enterprise tab, click the "Activate a license" button, and then paste in your license token. The page should change to show you that Enterprise features are now active.
 
+###Validating Your Enterprise Token
+
+Your Metabase instance will need to be able to access the internet (specifically `https://store.metabase.com/api/[token-id]`) in order to validate your token and grant access to the Enterprise feature set. 
+
+If you need to route outbound Metabase traffic through a proxy on your network, use the following command:
+
+`java -DproxySet = true -DproxyHost = [your proxy's hostname] -DproxyPort = [your proxy's port] -jar. \ enterprise_metabase.jar`
```
Comment (flamber) [inline docs/enterprise-guide/activating-the-enterprise-edition.md]: The command doesn't seem correct - there's a dot after -jar and there's a backslash without a newline. And I'm not sure how Java handles spaces around equal signs, but there's not spaces in other doc commands.

Comment (salsakran) [reply docs/enterprise-guide/activating-the-enterprise-edition.md]: hmmm.... think you're right. will fix it.

**Pull request merged**